lib/gh_proxy/cache.ex

```diff
@@ -0,0 +1,17 @@
+defmodule GHProxy.Cache do
+  require Logger
+
+  alias GHProxy.User, as: User
+
+  @storage Application.compile_env(:gh_proxy, Cache)[:cache_name]
```

Comment:
Если у вас плохой конфиг, эта строчка выживет при компиляции, потому что для `nil` определен (имплементирован) `Access`, а `[]` достает значения через него. `@storage` установится в `nil` и потом выстрелит в ногу спустя сто лет в рантайме. На такие мелочи обязательно нужно стараться обращать внимание: их можно тестами никогда не отловить, потому что там `env` другой, и если это не кэш, а какой-нибудь адрес вебхука, куда раз в день надо сходить данные сложить…

```suggestion
  @storage Application.compile_env!(:gh_proxy, Cache)[:cache_name]
```